xla/python/profiler/internal/python_hooks.h

```diff
@@ -26,17 +26,15 @@ limitations under the License.
 
 #include "absl/container/flat_hash_map.h"
 #include "absl/memory/memory.h"
-#include "pybind11/cast.h"
-#include "pybind11/pybind11.h"
-#include "pybind11/pytypes.h"
+#include "nanobind/nanobind.h"
 #include "xla/tsl/platform/macros.h"
 #include "xla/tsl/platform/types.h"
 #include "tsl/profiler/protobuf/xplane.pb.h"
 
 namespace xla {
 namespace profiler {
 
-namespace py = ::pybind11;
+namespace nb = nanobind;
```

Comment:
Google's style rules forbid defining namespace aliases in header files. In the .h file, please use nanobind::

(It was wrong before, I admit.)